### ISSUE TITLE
Magnet that pushes things up (not like hover thing)

### ISSUE THREAD

**Issue by Squibard** [studio-minus/ppg-bugs-and-requests#2734]
### Feature description

Like a magnet that pushes things upward or outward

### Feature purpose

for more variety in what we can use to build

### Additional context

https://user-images.githubusercontent.com/112280080/217043669-f98969ac-cc9a-405e-972c-c3bba62eb335.mp4

**Comment by username90001** [comment]
good idea

**Comment by Drogo-man** [comment]
You can make magnets repel in the context menu.

**Comment by Squibard** [comment]
> You can make magnets repel in the context menu.

i want one that pushes upwards

**Comment by DindinYT37** [comment]
Are you talking about a fan?

**Comment by Squibard** [comment]
> Are you talking about a fan/propeller?

yes but u put it under the object like i did in the video and it only pushes metal

**Comment by DindinYT37** [comment]
> > Are you talking about a fan/propeller?
> 
> yes but u put it under the object like i did in the video and it only pushes metal

Fan/propeller with metal detector?

**Comment by DindinYT37** [comment]
> > > > Are you talking about a fan/propeller?
> > > 
> > > 
> > > yes but u put it under the object like i did in the video and it only pushes metal
> > 
> > 
> > Rotated fan/propeller with metal detector?
> 
> but can be on all the time with editable distance

Rotated fan/propeller that was toggled on and powered?

**Comment by Squibard** [comment]
> > > > > Are you talking about a fan/propeller?
> > > > 
> > > > 
> > > > yes but u put it under the object like i did in the video and it only pushes metal
> > > 
> > > 
> > > Rotated fan/propeller with metal detector?
> > 
> > 
> > but can be on all the time with editable distance
> 
> Rotated fan/propeller that was toggled on and powered?

i guess
